### PR TITLE
Change persist layer

### LINKED ISSUE #2: Create persist layer
Do research about library that can be used for persist layer.

Maybe use libraries e.g. SQLAlchemy or same library that is used by Django.

### PR DESCRIPTION
**Pull request by liephat**
Closes #2

### CODE REVIEW

File: app/data/ops.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+from app.data.ids import create_id
+from app.data.models import Image, Label, Base
+
+
+class ImageDataHandler:
+
+    def __init__(self):
+        Config = ConfigParser()
+
+        self.session = create_session(Config.data_file())
```
Comment (joergrs) [inline app/data/ops.py:16]: session is created but never closed

https://docs.sqlalchemy.org/en/13/orm/session_api.html#sqlalchemy.orm.session.Session.close

File: app/__main__.py [hunk 2]
```diff
@@ -1,31 +1,21 @@
...
-
-
-class ImageData(Resource):
-
-    def get(self, uid):
-        return ImageDataReader.get_labels(uid)
-
-
-api.add_resource(ImageData, "/image/data/<int:uid>")
+ImageData = ImageDataHandler()
```
Comment (joergrs) [inline app/__main__.py:13]: Coding style: variable names should begin with lower-case character

File: app/data/ops.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+from app.config.parser import ConfigParser
+from app.data.ids import create_id
+from app.data.models import Image, Label, Base
+
+
+class ImageDataHandler:
+
+    def __init__(self):
+        Config = ConfigParser()
```
Comment (joergrs) [inline app/data/ops.py:14]: also here

File: classify_images.py [hunk 4]
```diff
@@ -16,14 +15,11 @@
 ResNet = ResNet(Config.model('resnet'), Config.labels('resnet'))
 ResNet.load()
 
-# Create data frame to save image labels
-data = pd.DataFrame(columns=['uid', 'file', 'labels'])
-data.index.name = 'id'
+# Create data handler
+ImageData = ImageDataHandler()
```
Comment (joergrs) [inline classify_images.py:19]: and here

File: app/data/ops.py (hunk 1 above)
Comment (liephat) [reply app/data/ops.py]: I fixed this issue by implementing an auto_session method which automatically closes the session. See the changes by the new commit.

**Review by joergrs — APPROVED** [review]

**Pull request merged**